packages/axios-extension/src/abap/adt-catalog/services/list-package-service.ts

```diff
@@ -0,0 +1,81 @@
+import { AdtService } from './adt-service';
+import type { AdtCategory, PackageInfo } from '../../types';
+import XmlParser from 'fast-xml-parser';
+
+/**
+ * ListPackageService implements ADT requests for fetching a list of available package names
+ * from ABAP backend system.
+ */
+export class ListPackageService extends AdtService {
+    /**
+     * @see AdtService.getAdtCatagory()
+     */
+    private static AdtCategory = {
```

Comment:
Start name with lowercase? It's not a type

Reply:
I forgot there is already a static getter method for accessing adtCategory and the default impl in the parent class throws error if it is not implemented. I will just rename to small case and not apply the bigger change we discussed.